Dockerfile

```diff
@@ -12,11 +12,11 @@ FROM wurstbrot/dsomm-yaml-generation as yaml
 FROM nginx:alpine
 
 RUN chown -R nginx:nginx /var/cache/nginx \
- && chown -R nginx:nginx /var/log/nginx \
- && touch /var/run/nginx.pid \
- && chown -R nginx:nginx /var/run/nginx.pid
+      && chown -R nginx:nginx /var/log/nginx \
+      && touch /var/run/nginx.pid \
+      && chown -R nginx:nginx /var/run/nginx.pid
 
-COPY --from=yaml ["/var/www/html/src/assets/YAML/generated/generated.yaml", "/usr/share/nginx/html/assets/YAML/generated/generated.yaml"]
+COPY --from=yaml ["/var/www/html/src/assets/yaml/generated/generated.yaml", "/usr/share/nginx/html/assets/yaml/generated/generated.yaml"]
```

Comment:
the folder is upper case

Reply:
I've changed this with https://github.com/devsecopsmaturitymodel/DevSecOps-MaturityModel/pull/282/commits/baca9d2b34318d91436321a3698d7a998e7fae9f to avoid random capitalisation in the paths as most anything regarding paths is lower case.

Reply:
If that is the blocker, i can revert that commit of course, but i did not find anything wrong while testing.

Reply:
Yes, please revert. 

In https://github.com/devsecopsmaturitymodel/DevSecOps-MaturityModel-data and users of DSOMM will have used `YAML`.